NativeApp/src/Android/AndroidAppBase.cpp

```diff
@@ -30,8 +30,12 @@
 //#include <android/log.h>
 //#include <android_native_app_glue.h>
 #include <android/native_window_jni.h>
+#include <Errors.hpp>
 //#include <cpu-features.h>
 
+// temporary flag
+#define LOG_NATIVE_APPLICATION false
```

Comment:
If it is temporary, should it be removed?
If this is for development only, it is better to use `#ifdef DILIGENT_DEVLOPMENT` instead of #if (LOG_NATIVE_APPLICATION)